Drawing/DrawList.cs

```diff
@@ -488,4 +508,53 @@ public T AddButton(string label, Action action, Vector2? buttonSize = null)
 
         return DrawListOwner;
     }
-}
\ No newline at end of file
+
+    public T AddIconButton(FontAwesomeIcon icon, Action action, string? hoverTooltip = null)
+    {
+        DrawActions.Add(() =>
+        {
+            if (ImGuiComponents.IconButton(icon))
```

Comment:
Should use the overload for IconButton that takes a ID string, or else there will be an ID conflict and only the first button will work.

Reply:
Is there a good way to define which ID to use? Should I just add an ID parameter to the signature of `AddIconButton`?